### PR TITLE
Set Esc shortcut globally for dialogs

### PR DESCRIPTION
**Pull request by everdimension**
Refactor some initialization code

### CODE REVIEW

File: src/ui/Onboarding/initialization.ts [hunk 1]
```diff
@@ -16,5 +16,13 @@ export async function maybeOpenOboarding() {
...
   const mode =
     hasOnboardingUrl || (!isPopup && !userHasWallets) ? 'onboarding' : 'wallet';
+  if (mode === 'onboarding' && !isPopup) {
+    // TODO: setting "context=onboarding" is duplicated here and in {openOnboarding(url)}
+    const searchParams = new URLSearchParams(window.location.search);
+    if (!searchParams.has('context')) {
+      searchParams.append('context', 'onboarding');
+      window.location.search = searchParams.toString();
+    }
+  }
```
Comment (vyorkin) [inline src/ui/Onboarding/initialization.ts:26]: Is this change related to the PR?

Comment (everdimension) [reply src/ui/Onboarding/initialization.ts:26]: No, just a bug I found because I planned to support the shortcut for this case, too, but turned out to require too many changes to onboarding redirect logic

File: src/ui/style/global.module.css [hunk 2]
```diff
@@ -83,6 +83,7 @@ body.pageLayout {
   --body-width: 100%;
   height: 100vh;
   width: 100%;
+  border: none;
```
Comment (vyorkin) [inline src/ui/style/global.module.css:86]: Not sure, but looks like it is not related to global shortcuts

**Review by vyorkin — APPROVED** [review]

File: src/ui/style/global.module.css (hunk 2 above)
Comment (everdimension) [reply src/ui/style/global.module.css:86]: right, it's a also a bug I found due to this refactoring: https://github.com/zeriontech/zerion-wallet-extension/pull/523

File: src/ui/App/App.tsx [hunk 3]
```diff
@@ -389,6 +393,35 @@ dayjs.extend(relativeTime);
...
+        />
+      ) : null}
+
+      <KeyboardShortcut
+        combination="ctrl+alt+0"
+        onKeyDown={() => {
+          // Helper for development and debugging :)
+          const url = new URL(window.location.href);
+          url.searchParams.set('windowContext', 'tab');
+          window.open(url, '_blank');
```
Comment (zerts) [inline src/ui/App/App.tsx:418]: While we are refactoring this part, let's change this to `openInFullScreen` (for Arc support)

Comment (everdimension) [reply src/ui/App/App.tsx:418]: "in tab view" you mean? ;)
